src/components/ProjectDetail/styles/StatusColumn.scss

```
@@ -1,64 +1,29 @@
 @import 'styles/variables';
 .statuscolumn {
    height: 100vh;
-   flex: 0 0 20%;
-   transform: rotateX(180deg);
+   min-width: 300px;
+   max-width: 300px;
+   margin-left: 10px;
+   margin-top: 15px;
+   padding: 10px;
+   border-radius: 5px;
+   border: 1px solid #d9dde1;
+   background-color: #efeff0;
+   transition: 0.2s linear;
    .statuscolumn__header {
-      font-size: 20px;
-      font-weight: bold;
+      display: block;
+      font-size: 16px;
+      line-height: 1.5;
+      font-weight: 600;
       color: $text;
-      text-align: center;
-      padding: 10px 0;
-   }
-   .statuscolumn__header-1 {
-      background-color: rgba(0, 0, 0, 0.1);
-   }
-   .statuscolumn__header-2 {
-      background-color: #a29bfe;
-   }
-   .statuscolumn__header-3 {
-      background-color: #fd79a8;
-   }
-   .statuscolumn__header-4 {
-      background-color: #6c5ce7;
-   }
-   .statuscolumn__header-5 {
-      background-color: #55efc4;
-   }
-   .statuscolumn__header-6 {
-      background-color: $error;
-   }
-   .statuscolumn {
-      margin-left: 5px;
-   }
-   .statuscolumn-1 {
-      border: 2px solid rgba(0, 0, 0, 0.7);
-      height: 100%;
-   }
-   .statuscolumn-2 {
-      border: 2px solid #a29bfe;
-      height: 100%;
-      border-left: none;
-   }
-   .statuscolumn-3 {
-      border: 2px solid #fd79a8;
-      border-left: none;
-      height: 100%;
-   }
-   .statuscolumn-4 {
-      border: 2px solid #6c5ce7;
-      height: 100%;
-      border-left: none;
-   }
-   .statuscolumn-5 {
-      border: 2px solid #55efc4;
-      height: 100%;
-      border-left: none;
-   }
-   .statuscolumn-6 {
-      border: 2px solid $error;
-      height: 100%;
-      border-left: none;
-      border-right: none;
+      text-align: left;
+      padding: 10px;
+   }
+   .statuscolumn__body {
+      width: 100%;
+      height: 200px;
+      max-height: 200px;
```

```suggestion
      height: 100%;
```

Bỏ cái này, e để height 200 với màn của e thì được chứ nếu màn hình có height lớn n chỉ scroll 1 tí chút thôi